anomalo/client.go

```diff
@@ -131,6 +131,8 @@ func (c *Client) Ping() (*PingResponse, error) {
 	return data, nil
 }
 
+// GetTableInformation looks up a table by `tableName`.
+// `tableName` must start with its warehouse name.
```

Comment:
```suggestion
// `tableName` must start with its warehouse name.
//
// For example, a Snowflake table with a warehouse called `square` and a table
// called `items.variations` should be referenced as `square.items.variations`.
```

Added an example to help clarify a bit